nx-X11/programs/Xserver/mi/Imakefile

```diff
@@ -179,6 +183,9 @@ NormalLibraryTarget(cbrt,cbrt.o)
 
 SpecialCObjectRule(miinitext,$(ICONFIGFILES),$(EXT_DEFINES))
 SpecialCObjectRule(miscrinit,$(ICONFIGFILES),$(EXT_DEFINES))
+SpecialCObjectRule(miglblt,$(ICONFIGFILES),$(FONT_DEFINES))
+SpecialCObjectRule(mipolytext,$(ICONFIGFILES),$(FONT_DEFINES))
```

Comment:
Hmm, no, it's not, in reality we're using a macro to redefine `GetGlyphs` to `dixGetGlyphs` so that we do not actually have to link to `libXfont`.